src/lib/pubsub/pubsub_build.c

```diff
@@ -276,7 +292,11 @@ pubsub_builder_finalize(pubsub_builder_t *builder)
   if (!dispatcher)
     goto err;
 
-  dispatch_fill_pub_binding_backptrs(builder, dispatcher);
+  pubsub_items_install_bindings(builder->items, dispatcher);
+  if (items_out) {
```

Comment:
`items_out != NULL` seems untested?

Reply:
277fe13952de33f6f24be5c5ace64e817610b405 now tests this case.